### ISSUE TITLE
pip install doesn't match github files

### ISSUE THREAD

**Issue by jcvall** [Kitware/seaborn_altair#7]
When I do a pip install seaborn_altair only categorial.py is installed.  No other files.  Is pypi up to date?

**Comment by jeffbaumes** [comment]
No, I had only pushed to pypi near the beginning of development, and the docs went ahead of what's released, sorry about that. When I did a quick check just now there seems to be some differences in altair's latest version that make errors pop up, so I wouldn't want to push to pypi in it's current state.

You are welcome to attempt a git checkout and `pip install -e .` and see what might work for you. As I have moved on from this for the most part, I'm afraid I can't be too much more help.
